library_generation/utils/utilities.sh

```diff
@@ -380,3 +375,17 @@ download_googleapis_files_and_folders() {
 get_gapic_generator_location() {
   echo "${GAPIC_GENERATOR_LOCATION}"
 }
+
+get_java_formatter_location() {
+  echo "${JAVA_FORMATTER_LOCATION}"
+}
+
+error_if_not_exists() {
+  local required_tool=$1
+  if [[ ! -f "${required_tool}" ]]; then
+    >&2 echo "File ${required_tool} not found in the "
+    >&2 echo "filesystem. Please configure your environment and store the "
+    >&2 echo "generator jar in this location"
```

Comment:
Since this logic is reused now, we should update the wording from "generator jar" to something more generic.

Reply:
Done.